lib/common/mobile/android/android-log-filter.ts

```diff
@@ -35,33 +38,19 @@ export class AndroidLogFilter implements Mobile.IPlatformLogFilter {
 		return data + EOL;
 	}
 
-	private getConsoleLogFromLine(lineText: string, pid: string): any {
-		// filter log line if it does not belong to the current application process id
-		if (pid && lineText.indexOf(pid) < 0) {
+	getConsoleLogFromLine(lineText: string, pid: string) {
+		if (pid && lineText.indexOf(pid) === -1) {
 			return null;
 		}
 
-		// Messages with category TNS.Native and TNS.Java will be printed by runtime to Logcat only when `__enableVerboseLogging()` is called in the application.
-		const acceptedTags = [
-			"chromium",
-			"Web Console",
-			"JS",
-			"ActivityManager",
-			"System.err",
-			"TNS.Native",
-			"TNS.Java",
-		];
-
-		let consoleLogMessage: { tag?: string; message: string };
-
+		let consoleLogMessage;
 		const match =
-			lineText.match(AndroidLogFilter.LINE_REGEX) ||
-			lineText.match(AndroidLogFilter.API_LEVEL_23_LINE_REGEX);
+			lineText.match(AndroidLogFilter.API_LEVEL_23_LINE_REGEX) ||
+			lineText.match(AndroidLogFilter.LINE_REGEX);
```

Comment:
Note: I flipped this condition to speed up matching by optimizing for API23 first and then falling back to legacy regex.